### DISCUSSION TITLE
求助！关于支付退款回调！

### DISCUSSION THREAD

**Discussion by shanquan954** [yansongda/pay#719] [Q&A]
我想要在支付和退款的回调接口中拿到信息，写到数据库，方便后续查询，但是因为本地环境，苦于调试数据，而且我是第一次编写支付模块，所以想问有没有可以参考的callback写法😥

**Comment by yansongda** [comment]
文档里不是有具体用法么

**Reply by shanquan954** [reply]
有的，我根据微信的文档可以得知我能获得什么数据，但是我....不太自信，毕竟写完不能用就白忙活了，如果有参考是最好的

**Reply by shanquan954** [reply]
好吧，是我太矫情了，也不是现在一定要写完，等上线我再慢慢搞吧...
